src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+
+.calendar {
+  width: calendarWidth($gap-size, $cell-size, $padding-size);
+  margin: 0 auto;
+  padding: $padding-size;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+
+  &__cell {
+    @include square($cell-size);
+    border: 1px solid black;
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+  }
+
+  @each $day, $day-order in $week {
+    &__start-day--#{$day} :first-child {
+      margin-left: $day-order * $cell-size
+        - $cell-size + $day-order - 2 * $gap-size;
+    }
+  }
+
+  &__cell:hover {
+    cursor: pointer;
+    background-color: pink;
+  }
+}
+
+@for $day from 1 to 32 {
```

Comment:
to 31?

Reply:
If I choose value 31, the amount of days is 30. Trying to fix it now